docs/orchestrating-elastic-stack-applications/kibana.asciidoc

```diff
@@ -150,12 +150,15 @@ spec:
     spec:
       containers:
       - name: kibana
+        env:
+          - name: NODE_OPTIONS
+            value: "--max-old-space-size=2048"
         resources:
           requests:
             memory: 1Gi
             cpu: 0.5
           limits:
-            memory: 2Gi
+            memory: 2.5Gi
```

Comment:
Not familiar with Node.js but how did we arrive at this 0.5Gi overhead?

Reply:
It was an educated guess on my part too. My reasoning was that if Node.js is allowed to grow the heap to 2Gi, there must be some slack left for the app to run properly.